### PR TITLE
bst project page

### PR DESCRIPTION
**Pull request by jaseweston**

### CODE REVIEW

File: projects/README.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ This directory also contains subfolders for some of the projects which are house
   
 ## Chit-chat
 
+- **Blended Skill Talk** [[project]](https://parl.ai/projects/bst/).
+  _Blending the skills of engagingness, personality, empathy and knowledge with a task that mixes PersonaChat, Empathetic Diaogues and Wizard of Wikipedia elements._
```
Comment (EricMichaelSmith) [inline projects/README.md:39]: nit: typo in "Empathetic Dialogues"

File: projects/bst/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## Blended Skill Talk
+
+### _Can You Put it All Together: Evaluating Conversational Agents' Ability to Blend Skills_
+
+Eric Michael Smith, Mary Williamson, Kurt Shuster, Jason Weston, Y-Lan Boureau
```
Comment (EricMichaelSmith) [inline projects/bst/README.md:5]: Ha, we can put the asterisks by my name and Mary's name :)

File: projects/bst/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<p align="center"><img width="90%" src="bst_fig.png" /></p>
+
+
+## Paper
+
+[Link](https://arxiv.org/abs/2004.08449)
+
+## Looking at the data
+
+You can view the data or train your own ParlAI agent on the LIGHT tasks with
```
Comment (EricMichaelSmith) [inline projects/bst/README.md:20]: I assume you mean the BST task, not the LIGHT task?

File: projects/README.md (hunk 1 above)
Comment (jaseweston) [reply projects/README.md]: fixed

File: projects/bst/README.md (hunk 2 above)
Comment (jaseweston) [reply projects/bst/README.md]: done

File: projects/bst/README.md (hunk 3 above)
Comment (jaseweston) [reply projects/bst/README.md]: ah yea sorry, fixed

**Pull request merged**